modules/codegen/src/smithy4s/codegen/internals/Renderer.scala

```diff
@@ -164,10 +164,42 @@ private[internals] class Renderer(compilationUnit: CompilationUnit) { self =>
       }
   }
 
+  /**
+    * Returns the given list of Smithy documentation strings formatted as Scaladoc comments
+    *
+    * @param l splitted docstring from the IR
+    * @return formatted list of scaladoc lines
+    */
+  private def makeDocLines(l: List[String]): List[Line] = {
+    @tailrec
+    def loop(l: List[String], acc: List[Line]): List[Line] = {
+      l match {
+        case Nil => acc
+        case hd :: tl if tl == Nil =>
+          loop(tl, acc :+ line"  * $hd" :+ line"  */")
```

Comment:
```suggestion
        case hd :: Nil =>
          loop(Nil, acc :+ line"  * $hd" :+ line"  */")
```

also, I guess you can inline the `loop` call to be just the second parameter (because `acc` would be returned):

```suggestion
        case hd :: Nil =>
          acc :+ line"  * $hd" :+ line"  */"
```

Reply:
I did this, but the `case Nil` is still there because else I get an exhaustion warning

Reply:
yep, I guess that'll have to stay